src/Compiler/Service/TransparentCompiler.fs

```diff
@@ -551,8 +551,8 @@ type internal TransparentCompiler
         let projectReferences =
             getProjectReferences projectSnapshot "ComputeTcConfigBuilder"
 
-        // TODO: script support
-        let loadClosureOpt: LoadClosure option = None
+        let loadClosureOpt: LoadClosure option =
+            backgroundCompiler.ScriptClosureCache.TryGet(AnyCallerThread, projectSnapshot.ToOptions())
```

Comment:
This might be good to get started but eventually we aim to get rid of `BackgroundCompiler` and `FSharpProjectOptions` completely.

Reply:
Thanks for the review.
So in the end, we should really have a cache defined like this?
```fsharp
MruCache<AnyCallerThreadToken, FSharpProjectSnapshot, LoadClosure>
```

Reply:
That would be better. Or you might want to try to unify it with all the other caches and use `AsyncMemoize`. 

`FSharpProjectSnapshot` could be quite big, in case of a lot of referenced projects. Depending on what invalidates the cache you might want something like `ProjectSnapshot.FullKey` or some of the other Keys it has.

Reply:
Makes sense, yeah. Will try :)